lib/licensee/projects/git_project.rb

```diff
@@ -44,7 +44,9 @@ def close
 
       def commit
         @commit ||= if revision
-          repository.lookup(revision)
+          commit = repository.rev_parse(revision)
+          commit = commit.target until commit.is_a?(Rugged::Commit)
```

Comment:
TIL'd `until` :smile:

Reply:
Wasn't my instinct, but Rubocop made me do it! Haha